### ISSUE TITLE
Autocomplete using ctrl+space does absolutely nothing (or sometimes skip a line)

### ISSUE THREAD

**Issue by hrqmonteiro** [neoclide/coc.nvim#2718]
## Result from CocInfo
```
## versions

vim version: NVIM v0.5.0-dev+944-g82100a6bd
node version: v14.13.1
coc.nvim version: 0.0.79-437fdaadab
coc.nvim directory: /home/hrq/.config/nvim/plugged/coc.nvim
term: xterm-256color
platform: linux

## Log of coc.nvim

2020-12-15T14:36:58.006 INFO (pid:11882) [services] - registered service "highlight"
2020-12-15T14:36:58.063 INFO (pid:11882) [language-client-index] - highlight started with 11898
2020-12-15T14:36:58.076 INFO (pid:11882) [services] - registered service "vimlsp"
2020-12-15T14:36:58.077 INFO (pid:11882) [services] - vim language server state change: stopped => starting
2020-12-15T14:36:58.080 INFO (pid:11882) [plugin] - coc.nvim 0.0.79-437fdaadab initialized with node: v14.13.1 after 169ms
2020-12-15T14:36:58.085 INFO (pid:11882) [language-client-index] - vimlsp started with 11905
2020-12-15T14:36:58.385 INFO (pid:11882) [services] - vim language server state change: starting => running
2020-12-15T14:36:58.390 INFO (pid:11882) [services] - service vimlsp started
```

## Describe the bug

I am trying to set `coc.nvim` for the first time, and i am watching many configs and trying to reproduce them. I want the suggestions box to have the first suggestion selected, and i want to be able to autocomplete using `Ctrl+Space`.

## Reproduce the bug

When i did :verbose inoremap <c-space> there was nothing attributed to it. And not that i configurated the `coc.vim`file, it returns me
```
i  <C-Space>   * coc#refresh()
        Last set from ~/.config/nvim/plug-config/coc.vim line 43
```

And i did `source $HOME/.config/nvim/plug-config/coc.vim`on my `init.vim`

So basically is all set correctly. But ctrl+space does nothing, it doesnt autocomplete when i start typing something and the suggestions box come up. And sometimes what it does is skip a line.

Also, on my coc.vim file, i have the basic configuration file from the README.

**Comment by chemzqm** [comment]
Could be your terminal not send that key codes to vim, use other key mapping instead,

**Comment by hrqmonteiro** [comment]
> Could be your terminal not send that key codes to vim, use other key mapping instead,

I use alacritty, there are no keybindings assigned to Ctrl+Space in that.

**Comment by chemzqm** [comment]
Debug your vim, doesn't seems issue of coc.nvim

**Comment by hrqmonteiro** [comment]
Is not an issue on my shell, you can check [here](https://github.com/alacritty/alacritty/issues/4575) if you want, neither on my vim, as you can see, i am able to pass a ctrl+space for other things in vim and it works, as i stated there.

It just won't work with coc and the coc#refresh command.

**Comment by chemzqm** [comment]
coc#refresh() does trigger completion only.

**Comment by aureooms** [comment]
@hrqmonteiro Out of curiosity, have you tried sourcing your `~/.vimrc` after `vim` is started?

**Comment by hrqmonteiro** [comment]
> @hrqmonteiro Out of curiosity, have you tried sourcing your `~/.vimrc` after `vim` is started?

Hi, i actually use Neovim, i don't have a `.vimrc` file.

**Comment by aureooms** [comment]
@hrqmonteiro Ok. I use `vim`. I tried to use the `<c-@>` shortcut but somehow it only works when I source the config file after vim has started. Before sourcing it I can see that the mapping is there, and that keystrokes get registered by vim (`^V^@` in insert mode), but the mapping does not "work".

**Comment by JulianH99** [comment]
c-space does not work for me on either windows terminal nor wezterm under windows, though weirdly it works on kitty on linux

**Comment by samrocksc** [comment]
i too have the issue, i disabled all my plugins, and attempted it, and then reinstalled coc-tsserver.  I think it may be with neovim itself somewhere.

**Comment by samrocksc** [comment]
@make-github-pseudonymous-again this method worked for me without the sourcing
